crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -745,6 +745,72 @@ pub fn layer_norm_config(node: &Node) -> (LayerNormConfig, bool) {
     )
 }
 
+/// Create a PadConfig from the attributes of the node
+pub fn pad_config(node: &Node) -> PadConfig {
+    fn get_pads(node: &Node) -> Vec<usize> {
+        if node.inputs.len() != 3 {
+            panic!("Pad: must provide three inputs")
+        }
```

Comment:
The says at least 2 required (so check should be  node.inputs.len() > 1) 

3rd and 4th inputs are optional.